docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -12,29 +12,58 @@ Office Add-ins are web applications that are displayed using iFrames when runnin
+|Android|any|Chrome|
+
+## Perpetual Office on Windows
+
+For perpetual Office on Windows, the browser that is used is determined by the operating system, the Office version, whether the license is consumer or commercial, and whether the Edge WebView2 (Chromium-based) is installed.
+
+|OS|Office version|Consumer vs. Commercial|Edge WebView2 (Chromium-based) installed?|Browser|
+|:-----|:-----|:-----|:-----|:-----|
+|Windows 7, 8.1, 10, 11 | Office 2013 | Doesn't matter |Doesn't matter|Internet Explorer 11|
+|Windows 7, 8.1, 10, 11 | Office 2016| Commercial</br>(Build number form is `16.0.xxxx.xxxxx`,</br> ending with two blocks of 4 digits, </br> e.g., 16.0.5197.1000.) |Doesn't matter|Internet Explorer 11|
+|Windows 7, 8.1, 10, 11 | Office 2019| Commercial</br>(Build number form is `1808 (xxxxx.xxxxxx)`,</br> ending with two blocks of 5 digits, </br> e.g., 1808 (Build 10388.20027).)   |Doesn't matter|Internet Explorer 11|
+|Windows 7, 8.1, 10, 11 | Office 2016 to Office 2019| Consumer (Build number form is `YYMM (xxxxx.xxxxxx)`,</br> ending with two blocks of 5 digits, </br> e.g., 2206 (Build 15330.20264).)  |No |Microsoft Edge<sup>1, 2</sup> with original WebView (EdgeHTML)|
+|Windows 7, 8.1, 10, 11 | Office 2016 to Office 2019|  Consumer (Build number form is same as preceding row.)  |Yes<sup>3</sup>|Microsoft Edge<sup>1</sup> with WebView2 (Chromium-based)|
+|Windows 10, 11 | Office 2021| Doesn't matter |Yes<sup>3</sup> |Microsoft Edge<sup>1</sup> with WebView2 (Chromium-based)|
+
+<sup>1</sup> When Microsoft Edge is being used, the Windows Narrator (sometimes called a "screen reader") reads the `<title>` tag in the page that opens in the task pane. When Internet Explorer 11 is being used, the Narrator reads the title bar of the task pane, which comes from the **\<DisplayName\>** value in the add-in's manifest.
+
+<sup>2</sup> If your add-in includes the **\<Runtimes\>** element in the manifest, then it will not use Microsoft Edge with the original WebView (EdgeHTML). If the conditions for using Microsoft Edge with WebView2 (Chromium-based) are met, then the add-in uses that browser. Otherwise, it uses Internet Explorer 11. For more information, see [Runtimes](/javascript/api/manifest/runtimes).
+
+<sup>3</sup> On Windows versions prior to Windows 11, the WebView2 control must be installed so that Office can embed it. It's installed with Microsoft 365, version 2101 or later, and with one-time purchase Office 2021 or later; but it isn't automatically installed with Microsoft Edge. If you have an earlier version of Microsoft 365 or one-time purchase Office, use the instructions for installing the control at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/). On Microsoft 365 builds before 16.0.14326.xxxxx, you must also create the registry key **HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Win32WebView2** and set its value to `dword:00000001`.
```

Comment:
```suggestion
<sup>3</sup> On Windows versions prior to Windows 11, the WebView2 control must be installed so that Office can embed it. It's installed with Microsoft 365, version 2101 or later, and with one-time purchase Office 2021 or later; but it isn't automatically installed with Microsoft Edge. If you have an earlier version of Microsoft 365 or a perpetual version of Office, use the instructions for installing the control at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/). On Microsoft 365 builds before 16.0.14326.xxxxx, you must also create the registry key **HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Win32WebView2** and set its value to `dword:00000001`.
```